crates/fj-core/src/services/validation.rs

```diff
@@ -3,16 +3,34 @@ use std::{collections::BTreeMap, error::Error, thread};
 use crate::{
     objects::{BehindHandle, Object},
     storage::ObjectId,
-    validate::ValidationError,
+    validate::{ValidationConfig, ValidationError},
 };
 
 use super::State;
 
 /// Errors that occurred while validating the objects inserted into the stores
-#[derive(Default)]
 pub struct Validation {
     /// All unhandled validation errors
     pub errors: BTreeMap<ObjectId, ValidationError>,
+    /// Optional validation config
+    config: Option<ValidationConfig>,
```

Comment:
It's probably simpler to instead have `config: ValidationConfig`, and initialize that with its default value, if not provided.

But this would be a nice-to-have. Perfectly fine to merge this as-is.

Reply:
This has also been addressed.